### PR TITLE
Use client-side CUSTOM_PEERS for Topo updater

### PR DESCRIPTION
**Pull request by rdlrt**

### CODE REVIEW

File: scripts/cnode-helper-scripts/topologyUpdater.sh [hunk 1]
```diff
@@ -130,10 +130,28 @@ else
...
 
-[[ -z "${CUSTOM_PEERS}" ]] && CUSTOM_PEERS_PARAM="" || CUSTOM_PEERS_PARAM="&customPeers=${CUSTOM_PEERS}"
-
 [[ ${TU_PUSH} = "Y" ]] && curl -s -f -4 "https://api.clio.one/htopology/v1/?port=${CNODE_PORT}&blockNo=${blockNo}&valency=${CNODE_VALENCY}&magic=${NWMAGIC}${T_HOSTNAME}" | tee -a "${LOG_DIR}"/topologyUpdater_lastresult.json
-[[ ${TU_FETCH} = "Y" ]] && curl -s -f -4 -o "${TOPOLOGY}".tmp "https://api.clio.one/htopology/v1/fetch/?max=${MAX_PEERS}&magic=${NWMAGIC}${CUSTOM_PEERS_PARAM}" && \
-mv "${TOPOLOGY}".tmp "${TOPOLOGY}"
-
+if [[ ${TU_FETCH} = "Y" ]]; then
+  curl -s -f -4 -o "${TOPOLOGY}".tmp "https://api.clio.one/htopology/v1/fetch/?max=${MAX_PEERS}&magic=${NWMAGIC}"
+  if [[ -z "${CUSTOM_PEERS}" ]]; then
```
Comment (Scitz0) [inline scripts/cnode-helper-scripts/topologyUpdater.sh:136]: Should be -n and not -z right?

File: scripts/cnode-helper-scripts/topologyUpdater.sh [hunk 2]
```diff
@@ -130,10 +130,28 @@ else
...
+      colons=$(echo "${p}" | tr -d -c ':' | awk '{print length}')
+      case $colons in
+        1) addr="$(cut -d: -f1 <<< "${p}")"
+           port=$(cut -d: -f2 <<< "${p}")
+           valency=1;;
+        2) addr="$(cut -d: -f1 <<< "${p}")"
+           port=$(cut -d: -f2 <<< "${p}")
+           valency=$(cut -d: -f3 <<< "${p}");;
+        *) echo "ERROR: Invalid Custom Peer definition '${p}'. Please double check CUSTOM_PEERS definition"
+           exit 1
```
Comment (Scitz0) [inline scripts/cnode-helper-scripts/topologyUpdater.sh:149]: missing `;;`

**Review by Scitz0 — CHANGES_REQUESTED** [review]
Some notes added. In addition to this we should make a change to `CNODE_HOSTNAME` to also handle empty/commented case. 
Replace: 
`if [ "${CNODE_HOSTNAME}" != "CHANGE ME" ]; then`
with:
if [ -n ${CNODE_HOSTNAME} && "${CNODE_HOSTNAME}" != "CHANGE ME" ]; then

File: scripts/cnode-helper-scripts/topologyUpdater.sh (hunk 1 above)
Comment (rdlrt) [reply scripts/cnode-helper-scripts/topologyUpdater.sh:136]: yip - my bad

**Review by Scitz0 — APPROVED** [review]
👍

**Pull request merged**